test/transaction_test.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Braintree.TransactionTest do
+  use ExUnit.Case, async: true
+
+  alias Braintree.Transaction
+
+  test "paypal transaction attributes are included as a map" do
+    transaction = %Transaction{
+      paypal: %{
+        payer_email: "nick@example.com"
+      }
+    }
+
+    assert transaction.id == nil
```

Comment:
Will you change this to `refute transaction.id` instead? After that this is good to go.